packages/did/src/Did.chain.ts

```diff
@@ -356,7 +356,7 @@ export function formatPublicKey(key: NewDidKey): PublicKeyEnum {
   return { [type]: publicKey }
 }
 
-function checkServiceEndpointInput(
+export function checkServiceEndpointInput(
```

Comment:
this was a quick hack. I don't know what the correct approach would be to share this method.